src/Fixer/Import/NoUnusedImportsFixer.php

```diff
@@ -114,9 +114,18 @@ function (NamespaceUseAnalysis $useDeclaration) use ($namespace) {
         }
     }
 
+    /**
+     * @param Tokens            $tokens
+     * @param NamespaceAnalysis $namespace
+     * @param array<int, int>   $ignoredIndexes
+     * @param string            $shortName
+     *
+     * @return bool
+     */
     private function importIsUsed(Tokens $tokens, NamespaceAnalysis $namespace, array $ignoredIndexes, $shortName)
     {
-        for ($index = $namespace->getScopeStartIndex(); $index <= $namespace->getScopeEndIndex(); ++$index) {
+        $namespaceEndIndex = $namespace->getScopeEndIndex();
```

Comment:
Is this assignment making the code more clear? Personally I'd put `$namespace->getScopeEndIndex()` directly in `for`, so it "makes pair" with `getScopeStartIndex` in single line.

Reply:
Actually I don't remember why I made this change. Maybe I wanted to reuse the variable but then I didn't. Or I wanted to avoid a method call on each `for` iteration. I can revert it if you think it's best.

Reply:
I would keep the variable to avoid calling `$namespace->getScopeStartIndex()` for each iteration so I mark this conversation as resolved. Let me know if you prefer reverting the change instead.